dotnet/src/Functions/Functions.OpenApi.Extensions/Extensions/ApiManifestKernelExtensions.cs

```diff
@@ -28,17 +28,17 @@ public static class ApiManifestKernelExtensions
     /// <param name="kernel">The kernel instance.</param>
     /// <param name="pluginName">The name of the plugin.</param>
     /// <param name="filePath">The file path of the API manifest.</param>
-    /// <param name="executionParameters">Optional execution parameters for the plugin.</param>
+    /// <param name="pluginParameters">Optional parameters for the plugin setup.</param>
     /// <param name="cancellationToken">Optional cancellation token.</param>
     /// <returns>The imported plugin.</returns>
     public static async Task<KernelPlugin> ImportPluginFromApiManifestAsync(
         this Kernel kernel,
         string pluginName,
         string filePath,
-        OpenApiFunctionExecutionParameters? executionParameters = null,
+        ApiManifestPluginParameters? pluginParameters = null,
```

Comment:
This (and the change below) look like a breaking change in the public API. Can we keep the original methods and add new methods with the new behaviour and have the old method call the new one?

Reply:
@markwallace-microsoft, I'd be happy to apply suggested changes, but I would like to double check before doing so: this extensions package hasn't been published yet. Do we have the same policy around breaking changes for unpublished packages? We would like to deprecate the old signature as the new signature better fits the spec.